Steer/DigitizerWorkflow/src/TPCDigitizerSpec.cxx

```diff
@@ -336,9 +341,10 @@ class TPCDPLDigitizerTask : public BaseDPLDigitizer
   o2::dataformats::MCTruthContainer<o2::MCCompLabel> mLabels;
   std::vector<o2::tpc::CommonMode> mCommonMode;
   bool mWriteGRP = false;
+  bool mWithMCTruth = true;
```

Comment:
this can be dropped, because you can check if a certain output is allowed and this would be enough for your purpose